site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -10,19 +10,19 @@ description: >
 This feature is in a pre-release state and might change.
 {{< /alert >}}
 
-There may be different types of clusters, such as on-premise, and Google Kubernetes Engine (GKE), used by a game to help with the cost saving and availability. For this purpose, Agones provides a mechanism to define priorities on the clusters. Priorities are defined on [GameServerAllocationPolicy](https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go) agones CRD. A matchmaker can enable the multi-cluster rules on a request and target [agones-allocator]({{< relref "allocator-service.md">}}) endpoint in any of the clusters and get resources allocated on the cluster with the highest priority. If the cluster with the highest priority is overloaded, the allocation request is redirected to the cluster with the next highest priority.
+There may be different types of clusters, such as on-premise, and Google Kubernetes Engine (GKE), used by a game to help with the cost saving and availability. For this purpose, Agones provides a mechanism to define priorities on the clusters. Priorities are defined on [GameServerAllocationPolicy](https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1/gameserverallocationpolicy.go) agones CRD. A matchmaker can enable the multi-cluster rules on a request and target [agones-allocator]({{< relref "allocator-service.md">}}) endpoint in any of the clusters and get resources allocated on the cluster with the highest priority. If the cluster with the highest priority is overloaded, the allocation request is redirected to the cluster with the next highest priority.
```

Comment:
Please replace the link with `ghlink`.

Reply:
What is ghlink?

Reply:
You can find examples of ghlink [here](https://github.com/googleforgames/agones/search?q=ghlink&unscoped_q=ghlink)

Reply:
Oh, is there some effort to convert links to ghlink? I was just following the local example doing what I thought was the minimum to make the tests pass. I'm going to be done with this PR now, so if it isn't sufficient, please close it.

Reply:
Thanks. It is good enough. I'll add the ghlink. Thanks for fixing the make test.